charts/gsp-cluster/templates/02-gsp-system/service-operator/crd.yaml

```diff
@@ -33,19 +33,147 @@ spec:
             aws:
               description: AWS specific subsection of the resource.
               properties:
-                diskSizeGB:
-                  type: integer
                 instanceType:
+                  description: InstanceType essentially defines the amount of memory
+                    and cpus on the database.
                   type: string
               type: object
+            secret:
+              description: Secret name to be used for storing relevant instance secrets
+                for further use.
+              type: string
           type: object
         status:
           description: PostgresStatus defines the observed state of Postgres
           properties:
+            events:
+              description: Events will hold more in-depth details of the current state
+                of the instance.
+              items:
+                description: Event is a single action taken against the resource at
+                  any given time.
+                properties:
+                  reason:
+                    description: Reason for the current status of the instance.
+                    type: string
+                  status:
+                    description: Status of the currently running instance.
+                    type: string
+                  time:
+                    description: Time of the event cast.
+                    format: date-time
+                    type: string
+                required:
+                - status
+                - time
+                type: object
+              type: array
+            id:
+              description: ID of an instance for a reference.
+              type: string
+            reason:
+              description: Reason for the current status of the instance.
+              type: string
+            status:
+              description: Status of the currently running instance.
+              type: string
+          required:
+          - id
+          - status
+          type: object
+      type: object
+  versions:
+  - name: v1beta1
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: sqs.queue.gsp.k8s.io
```

Comment:
I don't think we should put stuff in the `k8s.io` namespace

Reply:
Agreed. We should be using `govsvc.uk` probably.